nbformat/sign.py

```diff
@@ -20,7 +20,7 @@
     except ImportError:
         sqlite3 = None
 
-from ipython_genutils.py3compat import unicode_type, cast_bytes, cast_unicode
+from ipython_genutils.py3compat import cast_bytes, cast_unicode
```

Comment:
Are these still required in general as utility functions?

Reply:
Short answer is Yes. Longer answer is we could have a python3 only version of this that was simpler and rmove the need for py3compat. It required touching enough logic I delayed making that change until later.